convert2rhel/backup/files.py

```diff
@@ -150,6 +150,14 @@ def remove(self):
         except (OSError, IOError):
             loggerinst.debug("Couldn't remove restored file %s" % self.filepath)
 
+    def __eq__(self, value):
+        if hash(self) == hash(value):
+            return True
+        return False
```

Comment:
```suggestion
        return hash(self) == hash(value)
```

What about make it shorter?

Reply:
Was following the same style we have elsewhere